From 2f856812ae7bc705ba732812302ea6ccbe6e9b09 Mon Sep 17 00:00:00 2001
From: icg_intg <icg_intg@intel.com>
Date: Tue, 22 May 2018 15:43:23 +0800
Subject: [PATCH 1924/2328] media: intel-ipu4: css:
 scci_IPU4_master_20180517_2012 fw release

Message for Open Source:
New firmware css release.

Message for Internal:
[Feature]New firmware css release.
[Root Cause/Changes]
FW_Manifest_ID: 681a5db016ff4f538492e0856a91933fdabac0de
FW_Build_Branch: IPU4_master
FW_Build_Topic: auto_merge_queue
library version: commit-id 7de6eb4
FW Bundle patch number of this release :  329697
FW tag : scci_IPU4_master_20180517_2012

Change-Id: If7ea39d378336a22d366e8137c93d841350e354b
Depends-on: I981a27f23275b615a36a24dbbd32514f41f3155f
Depends-on: I5f5b5476f8e54a74ea724515a3c34b5cb96108e4
Depends-on: I364ca3d4a6ce3e6f12f4fa81e042b10beaf2261d
Signed-off-by: icg_intg <icg_intg@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h         | 2 +-
 .../lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h      | 2 +-
 .../lib2600psys/lib/psysapi/static/src/ia_css_psys_program_manifest.c | 4 ++--
 3 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h b/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
index 04e50b5..fc9e9f8 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
@@ -11,4 +11,4 @@
 * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
 * more details.
 */
-#define IA_CSS_FW_PKG_RELEASE  0x20180516
+#define IA_CSS_FW_PKG_RELEASE  0x20180517
diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
index 04e50b5..fc9e9f8 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
@@ -11,4 +11,4 @@
 * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
 * more details.
 */
-#define IA_CSS_FW_PKG_RELEASE  0x20180516
+#define IA_CSS_FW_PKG_RELEASE  0x20180517
diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/psysapi/static/src/ia_css_psys_program_manifest.c b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/psysapi/static/src/ia_css_psys_program_manifest.c
index 3fb92a0..be1ef96 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/psysapi/static/src/ia_css_psys_program_manifest.c
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/psysapi/static/src/ia_css_psys_program_manifest.c
@@ -1064,7 +1064,7 @@ void ia_css_program_manifest_init(
 /* WA for a visual studio compiler bug, refer to
  developercommunity.visualstudio.com/content/problem/209359/ice-with-fpfast-in-156-and-msvc-daily-1413263051-p.html
 */
-#pragma optimize( "", off )
+#pragma optimize("", off)
 #endif
 
 int ia_css_program_manifest_print(
@@ -1234,7 +1234,7 @@ int ia_css_program_manifest_print(
 
 #if defined(_MSC_VER)
 /* WA for a visual studio compiler bug */
-#pragma optimize( "", off )
+#pragma optimize("", off)
 #endif
 
 #endif
-- 
2.7.4

